otter/test/util/test_zk.py

```diff
@@ -36,13 +45,15 @@ class ZKCrudModel(object):
     """
     def __init__(self):
         self.nodes = {}
+        self.create_makepath = True
 
-    def create(self, path, content, makepath=False):
+    def create(self, path, value="", acl=None, ephemeral=False, sequence=False,
+               makepath=False):
         """Create a node."""
-        assert makepath is True, "makepath must be True"
+        assert makepath == self.create_makepath
```

Comment:
What is the purpose of making the caller pass this argument?  `assert`s make me nervous, because they're basically untested error conditions :)